hackathon_site/event/jinja2/event/dashboard_base.html

```diff
@@ -50,25 +72,28 @@ <h4 class="formH2"><b>Congratulations!</b> You've been accepted into {{ hackatho
 
                     {% if not rsvp_passed %}
                         <div style="display: flex; width: auto">
+                            {% if status == "Cannot Attend (Declined)" or status == "Accepted, awaiting RSVP" %}
```

Comment:
status accepted is already checked higher up, so `if application.rsvp is False or application.rsvp is None`